src/Common/Polyfills.cs

```diff
@@ -0,0 +1,11 @@
+namespace System.Linq
+{
+    public static class CustomEnumerableExtensions
+    {
+#if NET10_0_OR_GREATER
+        public static IEnumerable<T> Reverse<T>(T[] array) => Enumerable.Reverse(array);
+#else
+        public static IEnumerable<T> Reverse<T>(this T[] array) => Enumerable.Reverse(array);
```

Comment:
Note: This is needed to work around [a C# breaking change](https://github.com/dotnet/runtime/issues/107723) on non-net10.0 TFMs.